astropy/io/votable/tests/vo_test.py

```diff
@@ -718,7 +718,8 @@ def table_from_scratch():
 
 def test_open_files():
     for filename in get_pkg_data_filenames('data', pattern='*.xml'):
-        if filename.endswith('custom_datatype.xml'):
+        if (filename.endswith('custom_datatype.xml') or
+            filename.endswith('timesys_errors.xml')):
```

Comment:
Usually linter doesn't like the continuation of conditional at the same level as the nested statement.
```suggestion
                filename.endswith('timesys_errors.xml')):
```